content/en/network_monitoring/devices/_index.md

```diff
@@ -32,257 +32,17 @@ algolia:
-| Exagrid | [exagrid.yaml][65] <br> |
-| Extreme Networks | [extreme-switching.yaml][66] <br> |
-| [F5][207] | [f5-big-ip.yaml][67] <br> |
-| Fireeye | [fireeye.yaml][68] <br> |
-| [Fortinet][208] | [fortinet.yaml][70] <br>[fortinet-fortigate.yaml][71] <br> |
-| Gigamon | [gigamon.yaml][72] <br> |
-| [HP][209] | [hp-ilo4.yaml][73] <br>[_hp.yaml][74] <br>[hpe-proliant.yaml][75] <br> |
-| Huawei | [_huawei.yaml][76] <br> |
-| iXsystems | [ixsystems-truenas.yaml][77] <br> |
-| IBM | [ibm.yaml][78] <br> |
-| Infinera | [infinera-coriant-groove.yaml][79] <br> |
-| [Infoblox][213] | [infoblox-ipam.yaml][80] <br> |
-| [Juniper Networks][210] | [_juniper.yaml][81] <br>[juniper-ex.yaml][82] <br>[_juniper-junos-generic.yaml][83] <br>[juniper-mx.yaml][84]
-| Linksys | [linksys.yaml][87] <br> |
-| McAfee | [mcafee-web-gateway.yaml][88] <br> |
-| [Meraki][7] | [meraki-cloud-controller.yaml][89] <br>[meraki.yaml][90] <br> |
-| Mikrotik | [mikrotik-router.yaml][91] <br> |
-| Nasuni | [nasuni-filer.yaml][92] <br> |
-| NEC | [nec-univerge.yaml][93] <br> |
-| NetApp | [netapp.yaml][94] <br> |
-| Netgear | [netgear.yaml][95] <br> |
-| Nvidia | [nvidia.yaml][96] <br> |
-| Omron | [omron-cj-ethernet-ip.yaml][97] <br> |
-| Opengear | [_opengear.yaml][98] <br> |
-| [Palo Alto Networks][222] | [palo-alto.yaml][100] <br>[_palo-alto.yaml][101] <br> |
-| Peplink | [peplink.yaml][102] <br> |
-| [pfSense][212] | [pf-sense.yaml][103] <br> |
-| Raritan | [raritan-dominion.yaml][104] <br> |
-| Riverbed | [riverbed.yaml][105] <br> |
-| Ruckus | [ruckus.yaml][106] <br> |
-| Server Iron | [server-iron-switch.yaml][107] <br> |
-| Servertech | [servertech.yaml][108] <br> |
-| Silverpeak | [silverpeak-edgeconnect.yaml][109] <br> |
-| Sinetica | [sinetica-eagle-i.yaml][110] <br> |
-| Sophos | [sophos-xgs-firewall.yaml][111] <br> |
-| Synology | [synology-disk-station.yaml][112] <br> |
-| Tripplite | [tripplite.yaml][113] <br> |
-| Ubiquiti | [_ubiquiti.yaml][114] <br> |
-| Velocloud | [velocloud-edge.yaml][115] <br> |
-| Vertiv | [_vertiv.yaml][116] <br> |
-| VMware | [vmware-esx.yaml][117] <br> |
-| Watchguard | [watchguard.yaml][118] <br> |
-| Western Digital | [western-digital-mycloud-ex2-ultra.yaml][119] <br> |
-| Zebra | [zebra-printer.yaml][120] <br> |
-| Zyxel | [zyxel-switch.yaml][121] <br> |
+Network Device Monitoring(NDM) gives you visibility into your on-premises and virtual network devices, such as routers, switches, and firewalls. Automatically discover devices on any network, and start collecting metrics like bandwidth utilization, volume of bytes sent, and determine whether devices are up/down.
 
+{{< whatsnext desc="This section includes the following topics:">}}
+    {{< nextlink href="network_monitoring/devices/getting_started" >}}<u>Getting Started</u>: Getting started with Network Device Monitoring{{< /nextlink >}}
+    {{< nextlink href="network_monitoring/devices/supported_devicess" >}}<u>Supported devices</u>: View supported NDM devices{{< /nextlink >}}
```

Comment:
This link gives me a 404.

Reply:
oh Let me fix that, there is an extra "s" on the end oops